test/functional/fixtures/runner/using-shared-code/base-page.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>

```

Comment:
`index.html` is a better naming for this, because in fact it's the same as `the base page`